### DISCUSSION TITLE
Métricas, pesquisa e analytics

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#457] [Discussão]
Falaaa galera beleza?! Queria levantar essa discussão, sobre métricas, pesquisa e analytics, que ferramentas vocês usam? Oq costumam medir e como usam os dados coletados?!
Que ferramentas vocês indicam para colher métricas de aplicativos mobile?!

**Comment by raulfernando** [comment]
Boa, estava mesmo querendo saber informações a respeito sobre metricas

**Comment by woliveiras** [comment]
A @fabiolafaria manja de SEO e poderia ajudar nessa ein.

O @jpusp e @rafarbr são devs mobile, devem saber sobre boas plataformas pra acompanhar os apps. :+1:

**Comment by kakuzops** [comment]
Eu entrei na discussão ontem com um cara do meu trabalho sobre as métricas queria me aprofundar no assunto
